app/views/admin/conferences/edit.html.haml

```diff
@@ -11,6 +11,7 @@
       = f.input :short_title, hint: "A short title, e.g. 'oSC14', to be used in URLs"
       = f.input :description, hint: markdown_hint('A description of the conference.'), input_html: { rows: 5, data: { provide: 'markdown-editable' } }
       = f.input :color, hint: 'The color will be used eg for the dashboard.', input_html: {size: 6, type: 'color'}
+      = f.input :ticket_layout, as: :select, collection: Conference.ticket_layouts.keys, hint: "Layout type for tickets of the conference."
```

Comment:
I would exchange the ticket layout and the conference picture, I mean moving ticket layout down as the conference picture is more important :wink: